Week1/homework/meetup/meetup.js

```diff
@@ -0,0 +1,63 @@
+
+const mysql = require('mysql');
+
+// Require the data module
+const data = require('./data.js');
```

Comment:
I like the separation of concern here. Well done!

Reply:
Thank you so much